tests/playwright/pages/wp-admin-page.ts

```diff
@@ -124,7 +124,7 @@ export default class WpAdminPage extends BasePage {
 	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
 	 */
 	async setExperiments( experiments: {[ n: string ]: boolean | string } ) {
-		await this.page.goto( '/wp-admin/admin.php?page=elementor#tab-experiments' );
+		await this.page.goto( `/wp-admin/admin.php?page=settings#tab-experiments` );
```

Comment:
Can you please replace the backticks with single quotes?
```suggestion
		await this.page.goto( '/wp-admin/admin.php?page=settings#tab-experiments' );
```